book/reflow.md

```diff
@@ -723,7 +722,7 @@ Next up: the `BlockLayout`. Here `compute_height` looks like this:
 ``` {.python}
 class BlockLayout:
     def compute_height(self):
-        self.h = self.pt + self.pb
+        self.h = 0
```

Comment:
Is this correct, or was the book correct and the lab wrong?

Reply:
I think the height should include the padding here. It's a little weird that we add the top and bottom padding at the start. So the original is, I think, correct.